_docs/best-practices.md

```diff
@@ -185,6 +195,8 @@ s:dateCreated: "2016-12-13"
 s:license: https://www.apache.org/licenses/LICENSE-2.0
 ```
 
+### WDL
```

Comment:
Image is better. 
Come to think of it, it would be nice to nest the images under CWL and link to the example so that users can look at it directly. Also create or find a professional looking WDL example, get images, and link to that as well.